homeassistant/components/switch/rpi_rf.py

```diff
@@ -101,22 +105,23 @@ def is_on(self):
         """Return true if device is on."""
         return self._state
 
-    def _send_code(self, code, protocol, pulselength):
-        """Send the code with a specified pulselength."""
-        _LOGGER.info("Sending code: %s", code)
-        res = self._rfdevice.tx_code(code, protocol, pulselength)
-        if not res:
-            _LOGGER.error("Sending code %s failed", code)
-        return res
+    def _send_code(self, code_list, protocol, pulselength):
+        """Send the code(s) with a specified pulselength."""
+        _LOGGER.info("Sending code(s): %s", code_list)
+        for code in code_list:
+            self._rfdevice.tx_code(code, protocol, pulselength)
```

Comment:
Is there a reason you removed the return value? Seems like [rpi_rf has a meaningful return value](https://github.com/milaq/rpi-rf/blob/master/rpi_rf/rpi_rf.py#L90) .